docs/topics/mta-web-creating-jira-issues-for-migration-wave.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-web-creating-jira-issues-for-migration-wave_{context}"]
+= Creating Jira issues for a migration wave
+
+You can use a migration wave for creating Jira issues automatically for each application assigned to it.
+
+.Procedure
+
+. In *Migration* view, click *Migration waves*.
+. From the *Options* menu (three dots) of the migration wave for which you want to create Jira issues, select *Export to Issue Manager*.
++
+The *Export to Issue Manager* window opens.
+. Select the instance type (Jira Cloud or Jira Server/Datacenter).
+. Select the instance, project and issue type from the drop-down lists.
+. Click *Export*.
++
+The status of the migration wave on the *Migration waves* page changes to `Issues Created`. To see the status of each individual application of a migration wave, click the *Status* column.
++
+A separate Jira issue has been created for each application associated with the migration wave. The following fields of each issue are filled in automatically:
+
+* Title: `Migrate <app name>`
```

Comment:
Looking at the guide, we use "application" much more often than "app," but you know the context better than I do.